core-web/libs/sdk/experiments/src/lib/utils/utils.ts

```diff
@@ -101,3 +106,37 @@ export const dotLogger = (msg: string, isDebug?: boolean): void => {
         console.warn(`[dotCMS Experiments] ${msg}`);
     }
 };
+
+/**
+ * Sets the flag indicating that the experiment has already been checked.
+ *
+ * @function setFlagExperimentAlreadyChecked
+ * @returns {void}
+ */
+export const setFlagExperimentAlreadyChecked = (): void => {
+    sessionStorage.setItem(EXPERIMENT_ALREADY_CHECKED_KEY, 'true');
+};
```

Comment:
Why don't you just centralize all the storage in one class? indexdb and session will be handle in the same class, a "CRUD" kinds of thing service.

Reply:
Yes. Moved. Tnks.